.github/workflows/vllm_ascend_test_pr_light.yaml

```diff
@@ -168,3 +168,55 @@ jobs:
       runner: linux-aarch64-a2
       image: swr.cn-southwest-2.myhuaweicloud.com/base_image/ascend-ci/cann:8.3.rc2-910b-ubuntu22.04-py3.11
       type: light
+
+  e2e-accuracy:
+    needs: [lint, changes]
```

Comment:
I'm ok with this. one thought is should accuracy trigger for all jobs? @wangxiyuan or just in full test?